backside/README.md

```diff
@@ -0,0 +1,13 @@
+Setup:
+
+    dep init
```

Comment:
`dep` is deprecated. Please switch to go modules: https://blog.golang.org/using-go-modules

What you need to do is to create a module inside the backside directory with:
```
go mod init github.com/sk8mate/sk8/backside
```

and then commit `go.mod` and `go.sum` files to the repository. Also, you need to delete the `Gopkg.toml` and `Gopkg.lock` files (after initializing the module). Shortly, the `go.mod` is replacement for `Gopkg.toml` and `go.sum` is replacement for `Gopkg.lock`.